packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/CreateRulesTests.java

```diff
@@ -368,17 +380,34 @@ public void createRuleWithActions()
         final UserModel admin = dataUser.getAdminUser();
 
         final RestRuleModel rule = restClient.authenticateUser(admin).withPrivateAPI().usingNode(ruleFolder).usingDefaultRuleSet()
-                .createSingleRule(rulesUtils.createVariousActions());
+                .createSingleRule(rulesUtils.createRuleWithVariousActions());
 
         RestRuleModel expectedRuleModel = rulesUtils.createRuleModelWithDefaultValues();
-        expectedRuleModel.setActions(rulesUtils.createVariousActions().getActions());
+        expectedRuleModel.setActions(rulesUtils.createRuleWithVariousActions().getActions());
         expectedRuleModel.setTriggers(List.of("inbound"));
 
         restClient.assertStatusCodeIs(CREATED);
         rule.assertThat().isEqualTo(expectedRuleModel, ID, IS_SHARED)
                 .assertThat().field(IS_SHARED).isNull();
     }
 
+    /**
+     * Check we can create a rule with check in action with empty description parameter.
```

Comment:
What's meant to happen to the empty description?  Please could you add an assertion for the parameters that we should get back?

Reply:
I am working on that one - this should fail (after what we recently discussed and agreed upon). I think I merged the master branch before that change was added there.